restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java

```diff
@@ -2419,10 +2438,10 @@ else if (CallManagerResponse.class.equals(klass) && !((CallManagerResponse)messa
                     }
                     if (call != null && callInfo != null) {
                         create = new CreateCall(e164(callerId(verb)), e164(callee), username, password, false, timeout(verb),
-                                CreateCallType.SIP, accountId, callInfo.sid(), statusCallback, statusCallbackMethod, statusCallbackEvent, mediaAttributes);
+                                CreateCallType.SIP, accountId, callInfo.sid(), statusCallback, statusCallbackMethod, statusCallbackEvent, mediaAttributes, null);
```

Comment:
i thouhgt customerHeaders was alredy supported  for SIP noun..., i expected to pass custome headers here as well... or is it taken from uri later? Seems different ways to pass custom headers in CreateCall msg

Reply:
@jaimecasero Right, for SIP URI already supports custom headers and seems too much to extract the custom headers from the SIP URI only to add them to the SIP URI later

Reply:
why dont we pass customer headers for clients/number the same way, as uri params in to param, so header parsing is centralized in one place...

Reply:
@jaimecasero initially implementation was this way, but I reverted this because of https://telestax.atlassian.net/browse/RESTCOMM-1852.

If the `Client` or `Number` in the `CreateCall` request contains the custom headers, for example, `alice?X-custom-header1=1234` or `+151298989898?X-custom-header1=1234`, any extension that will be executed in the call flow will have to normalize the client/number and I strongly believe this is not responsibility of the extension.
Also, CallManager `outbound` method in case of Client, will have to normalize string to get the client.

For the above reasons, I decided its better to extract custom headers before passing the request to CallManager